### ISSUE TITLE
Loader - Accessibility violation in the 'inside buttons' story

### ISSUE THREAD

**Issue by sianford** [Sage/carbon#4644]
### Current behaviour

There is an accessibility violation against the `inside button` story stating "Ensures buttons have discernible text". Full violation details are:

Critical - Element does not have inner text that is visible to screen readers
Serious - aria-label attribute does not exist or is empty
Serious - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Serious - Element has no title attribute
Minor - Element's default semantics were not overridden with role="none" or role="presentation"

<img width="885" alt="Loader_insideButtons_accessibilty_violation" src="https://user-images.githubusercontent.com/12828099/146085615-01389f22-36d6-4bc5-80b2-535e789184bc.png">

### Expected behaviour

There are no accessibility violations.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/loader--inside-buttons

### JIRA Ticket (Sage Only)

FE-4635

### Suggested Solution

Add the relevant aria-label or aria-labelledby attributes to resolve the issue.

### Carbon Version

102.4.0

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.
